Formula/prs.rb

```diff
@@ -0,0 +1,32 @@
+class Prs < Formula
+  desc "Secure, fast & convenient password manager CLI with GPG & git sync"
+  homepage "https://timvisee.com/projects/prs"
+  url "https://github.com/timvisee/prs/archive/v0.4.1.tar.gz"
+  sha256 "f7f8f5d815cf1c4034f1c2aa36ed29b7e3ab2a884791b4b69ffc845d5d111524"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "gpgme"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@3"
+  end
+
+  def install
+    cd "cli" do
+      system "cargo", "install", *std_cargo_args
+    end
+
+    system "#{bin}/prs", "internal", "completions", "--output=completions/", "bash", "fish", "zsh"
+    bash_completion.install "completions/prs.bash"
+    fish_completion.install "completions/prs.fish"
+    zsh_completion.install "completions/_prs"
+  end
+
+  test do
+    system "#{bin}/prs", "help"
+    system "#{bin}/prs", "--store=.pass", "init"
+    system "#{bin}/prs", "--store=.pass", "list"
```

Comment:
Could we check some output/results here?
```suggestion
    system bin/"prs", "help"
    system bin/"prs", "--store=.pass", "init"
    system bin/"prs", "--store=.pass", "list"
```

Reply:
I'm afraid that asserting command output here is not stable and quite tricky.

All commands fail with a non-zero exit code if any error occurs. Is that enough?

`prs --version` outputs `prs-cli v0.4.1`, which could be asserted but I'm not sure if that's worth anything?

Actually storing and reading secrets in a password store requires generating a GPG key and adding it to the system keyring amongst other things. I therefore don't test that.

Reply:
Does any of this create a file or directory? Because we could assert that too.

Reply:
Running `prs --store=.pass init` creates `.pass/`, but this is already asserted by `prs --store=.pass list` internally.

Reply:
And what is the output of `prs list` when the directory does exist?

Reply:
Currently nothing, since no secret is stored during testing (because that requires a GPG key in the systems keychain).

Reply:
Here's what the commands in the test suite output:

![image](https://user-images.githubusercontent.com/856222/212688989-407cd66a-906d-4cdd-a1cd-f4cd4de1e3d1.png)

But again, this output isn't stable.

Reply:
Sounds like we can do `assert_equals "\n", shell_output("#{bin}/prs --store=.pass list")` since it'll never have any secrets in the temporary test directory.

Reply:
I'll add that!

Reply:
Added, with a `--quiet` flag to guarantee this is empty in future updates too.